src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1800,3 +1791,6 @@ should_have_the_first_letter_capitalized=
 Remember_password?=
 
 shared=
+
+Tools=
+BibTex=
```

Comment:
BibTeX is always spelled BibTeX - see https://github.com/koppor/jabref/issues/73

BibTeX is a name like "OS/2" or "Linux" or "LaTeX" and must not be translated.